src/run.ts

```diff
@@ -10,10 +10,15 @@ const LEDGER_SECRET =
 const LEDGER_SCALE = 6
 const CONNECTOR_URL = process.env.CONNECTOR_URL || 'http://localhost:7771' // where the connector settlement api is NOTE, NOT THE ACCOUNTS API OR THE BTP API
 const ENGINE_PORT = process.env.ENGINE_PORT || 3000 // Where to listen for connections on
+
+const REDIS_URI = process.env.REDIS_URI
```

Comment:
Is there any reason to have both `REDIS_URI` and separate `REDIS_HOST` and `REDIS_PORT` fields?

Reply:
It is meant to be temporary to be backwards compatible for people who were using `REDIS_HOST` and `REDIS_PORT`. Didn't want them to sudden have everything break if they had configs setup already as we don't have semantic versioning setup yet.